pkg/executor/process.go

```diff
@@ -17,7 +17,10 @@ import (
 // LoggedExecuteInDir will put wrapped JSON output to stdout AND get RAW output into out var
 // json logs can be processed later on watch of pod logs
 func Run(dir string, command string, envMngr secret.Manager, arguments ...string) (out []byte, err error) {
-	obfuscatedArgs := envMngr.Obfuscate([]byte(strings.Join(arguments, " ")))
+	obfuscatedArgs := []byte{}
+	if envMngr != nil {
+		obfuscatedArgs = envMngr.Obfuscate([]byte(strings.Join(arguments, " ")))
```

Comment:
Do we need to read env vars - envMngr.GetVars(envMngr.Variables) ?

Reply:
That part is done in every executor before Run() is called to execute the test - except for when nil is passed in in the init runner